src/test/directives/class-map_test.ts

```diff
@@ -81,14 +81,24 @@ suite('classMap', () => {
     assert.isFalse(el.classList.contains('foo'));
   });
 
-  test('can override static classes', () => {
```

Comment:
This behavior was never released so it's safe to change. With the implementation in this PR, static classes are always present, even if they're apparently overridden in the classMap. I think this behavior is fine - it should be very rare that a class is in both static and dynamic sets, but if someone was doing this then changing it would have been a breaking change.